libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -0,0 +1,72 @@
+import { getConfig } from '../../utils/utils.js';
+
+async function getKey(product) {
+  const config = getConfig();
+  let keyMatch = [];
+  const resp = await fetch(`${config.contentRoot ?? ''}/branch-io-key.json`);
+  if (resp.ok) {
+    const json = await resp.json();
+    keyMatch = json.data.filter(
+      (p) => p.product === product,
+    );
+  }
+  return keyMatch[0]?.key;
+}
+
+/* eslint-disable */
+function branchInit(header, key) {
+  let initValue = false;
+  function initBranch() {
+    if (initValue) {
+      return;
+    }
+    initValue = true;
+    (function (b, r, a, n, c, h, _, s, d, k) {
+      if (!b[n] || !b[n]._q) {
+        for (; s < _.length;) c(h, _[s++]);
+        d = r.createElement(a);
+        // d.async = 1;
+        d.src = 'https://cdn.branch.io/branch-latest.min.js';
```

Comment:
I guess this snippet is from Branch. Would it be possible to use our tools `loadScript` to load the code?

Reply:
Discussed this with @narcis-radu and we are ok with currently branch provided script for now. Hence marking this as resolved.